src/prefect/futures.py

```diff
@@ -79,7 +81,7 @@ def result(
         If the task run has not completed, this method will wait for the task run to complete.
 
         Args:
-            - timeout: The maximum number of seconds to wait for the task run to complete.
+            timeout: The maximum number of seconds to wait for the task run to complete.
             If the task run has not completed after the timeout has elapsed, this method will return.
             - raise_on_failure: If `True`, an exception will be raised if the task run fails.
```

Comment:
nit, for consistency
```suggestion
            raise_on_failure: If `True`, an exception will be raised if the task run fails.
```